TYPO3.Media/Classes/TYPO3/Media/Domain/Repository/AssetRepository.php

```diff
@@ -278,4 +284,13 @@ public function findAllIterator()
             ->where('a NOT INSTANCE OF TYPO3\Media\Domain\Model\ImageVariant')
             ->getQuery()->iterate();
     }
+
+    /**
+     * @param AssetInterface $object
+     */
+    public function remove($object)
+    {
+        $this->assetService->validateRemoval($object);
+        parent::remove($object);
+    }
```

Comment:
now this enforces the policy that usages cannot be removed, is there any need to be able to force it? since there's no way to remove the usages easily..